rebar.config

```
@@ -84,9 +84,9 @@
                 {runtime_tools, load},
                 {tools, load},
                 {logger_logstash_formatter, load},
-                sasl,
+                {woody_api_hay, load},
```

В моём представлении всё должно магическим образом заработать, если
```suggestion
                woody_api_hay,
```

---

Да, нашел тоже. Спасибо! Но я бы сделал более толерантным запуск https://github.com/rbkmoney/woody_api_hay/pull/3